modules/geotiff/src/lib/load-geotiff.ts

```diff
@@ -1,5 +1,4 @@
-import { fromUrl, fromBlob } from 'geotiff';
-import type { GeoTIFF } from 'geotiff';
+import { fromUrl, fromBlob, GeoTIFF } from 'geotiff';
```

Comment:
I think you might need to reformat? I thought the prettier config didn't have spaces around imports...

Reply:
For some reason `yarn lint fix` only fixes `module/<name>/test` and doesn't seem to touch `module/<name>/src`. The same is happening in zarr. Any thoughts?

Reply:
@ibgreen ?

Reply:
:$ Yes, adding `ts` to the linting/prettier rules in `.ocularrc` generated a ton of issues so I commented it out. 

I just did a pass and opened a PR that enabled typescript linting: https://github.com/visgl/loaders.gl/pull/1471

For some reason CI does not seem to run, so not sure it is ready to land.

Reply:
landed, please go ahead and rebase. If you get a lot of conflicts in the rebase due to prettier changes on master in the zarr module you should be able to just replace the module code with your branch and rerun prettier, perhaps like

```
rm -fr modules/zarr
git co manst/tiff-loader -- modules/zarr
yarn lint fix
```

Reply:
done!